doc/index.rst

```diff
@@ -1,12 +1,19 @@
+.. Orange3-Associate documentation master file, created by
+   sphinx-quickstart on Fri Jun  3 12:00:36 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
 
-Orange-Associate scripting documentation
-========================================
+Welcome to Orange3-Associate documentation!
+===========================================
+
+Widgets
+-------
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
-.. automodule:: fpgrowth
-   :members:
+   widgets/associationrules.rst
+   widgets/frequentitemsets.rst
```

Comment:
I think so. You can build it (you are building it, right? – that's why you include _make.bat_, right?) and inspect the resulting html files in _\_build_.